Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2469 -- Display group/division info on uploaded items

Resolves [EREGCSC-2469](https://jiraent.cms.gov/browse/EREGCSC-2469)

**Description**

Display Group and Division information for uploaded documents if it exists.  If this information exists, it should be displayed on uploaded documents when viewed on the Search page, the Subjects page, and the Reader View.

[Figma design](https://www.figma.com/file/43zWIEFLznumR7JCyNPpQK/Divisions?node-id=15%3A6194&mode=dev)

**This pull request changes:**

- Creates `DivisionLabel` Vue component
   - Group/Division abbreviations are shown by default
   - Hovering the cursor over these abbreviations will show a browser tooltip that contains the full names
- if Group/Division information exists, conditionally renders `DivisionLabel` component in the following places:
   - `PolicyResults` on both Search and Subjects page
   - `SupplementalContentObject` component in the reader view's right sidebar
- Updates Cypress tests and adds a snapshot test for `DivisionLabel`

**Steps to manually verify this change:**

1. Visit the [experimental deployment](https://8ip63xk6pd.execute-api.us-east-1.amazonaws.com/dev1202)
2. Visit the `admin` panel and make sure at least one uploaded document has a Group and Division attached to it
3. Ensure that the label looks as expected on the [Search page](https://8ip63xk6pd.execute-api.us-east-1.amazonaws.com/dev1202/search/?q=faraway), the [Subjects page](https://8ip63xk6pd.execute-api.us-east-1.amazonaws.com/dev1202/subjects/?type=internal), and the [Reader View](https://8ip63xk6pd.execute-api.us-east-1.amazonaws.com/dev1202/42/433/Subpart-A/2024-02-23/#433-11)
